scripts/zones/Nyzul_Isle/instances/nashmeiras_plea.lua

```diff
@@ -35,15 +35,15 @@ function onInstanceProgressUpdate(instance, progress)
     if (progress == 4) then
         local chars = instance:getChars()
 
-        for i, v in pairs(chars) do
-            v:startEvent(203)
-        end
-
         DespawnMob(ID.mob[59].RAZFAHD, instance)
         SpawnMob(ID.mob[59].ALEXANDER, instance)
 
     elseif(progress == 5) then
         instance:complete()
+		v:setPos(126.6543, 0, 20.1798, 0, 72)
+		for i, v in pairs(chars) do
+			v:startEvent(203)
```

Comment:
Be careful not to introduce full or half-tabs, only 4x spaces for indentation please